docs/feature-flag-rollout-guide.md

```
@@ -0,0 +1,266 @@
+WHERE is_generating = TRUE;
+```
+
+### translation_priority_queue
+
+**有効化後に確認:**
+- TTFT（Time To First Translation）の改善
+- IMMEDIATE優先度リクエストの処理時間
+- キャンセル率
+
+**アプリケーションログで確認:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)